cs/cli/vw_example.cpp

```diff
@@ -139,7 +139,7 @@ namespace VW
 
     System::String^ FormatFeature(vw* vw, feature_value& f1, feature_index& i1)
     {
-        uint64_t masked_weight_index1 = i1 & vw->reg.weight_mask;
+		uint64_t masked_weight_index1 = i1 & vw->wv->mask();
```

Comment:
Maybe we should have a different name than 'wv'?  First, it's easy to transpose with 'vw' and second I'm not sure that 'vector' is a good descriptor for the general concept you are encapsulating.  Maybe 'parameters'?